jormungandr-watchdog/src/service/stats.rs

```diff
@@ -1,53 +1,62 @@
+use atomig::Atomic;
+use std::sync::atomic::{AtomicU64, Ordering};
+
 /// keep some stats based on [Welford's online algorithm]
 ///
 /// [Welford's online algorithm]: https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Welford's_online_algorithm
 pub struct Stats {
-    count: u64,
-    mean: f64,
-    m2: f64,
+    count: AtomicU64,
+    mean: Atomic<f64>,
+    m2: Atomic<f64>,
```

Comment:
No need to add an extra deps for this. All needed is [f64::to_bits](https://doc.rust-lang.org/stable/std/primitive.f64.html#method.to_bits) and [f64::from_bits](https://doc.rust-lang.org/stable/std/primitive.f64.html#method.from_bits) and you store the value in an `AtomicU64`.